src/api/index.ts

```diff
@@ -20,7 +22,22 @@ const getLocalStorage: GetLocalStorage = (key, errorMsg = 'Something went wrong'
   }
 }
 
-export const requestNotes = () => new Promise(getLocalStorage('notes'))
+type GetUserNotes = () => (resolve: PromiseCallbackFun, reject: PromiseCallbackFun) => void
+const getUserNotes: GetUserNotes = () => (resolve, reject) => {
+  const notes = localStorage.getItem('notes')
```

Comment:
I think we should try to add types to variable declarations.

Reply:
Do you mean to the `notes` variable? Would `const notes: any = ...` do it?

Reply:
I think I mean the return value of getItem like `string | null`